src/main/resources/messages/sendMail.xml

```diff
@@ -106,6 +108,16 @@
             </else>
         </filter>
         <filter
+                xpath="get-property('uri.var.contentType') = '' or  not(string(get-property('uri.var.contentType')))">
+            <then>
+                <property name="uri.var.contentType" value=""/>
+            </then>
+            <else>
+                <property name="gmailParameters" expression="fn:concat(get-property('gmailParameters'),
+               'content-Type:', get-property('uri.var.contentType'),'&#xA;')"/>
+            </else>
+        </filter>
+        <filter
```

Comment:
This is not added with this. However does not be executed at any time? not(string(get-property('uri.var.messageBody')))